File: FrontEnd/src/app/features/dashboard/provider/interfaces/service/service-response.ts
export interface ServiceResponse {
  id: string;
  title: string;
  description: string;
  price: number;
  imagesUrl: string;
  websiteUrl: string;
  categoryName: string;
  providerId: string;
  status: string;
  createdAt: string;
  updatedAt?: string;
  
  showDetails?: boolean;
  isEditing?: boolean;
  editData?: any;
}

File: AuthTemplate-master/Ecommerce.Entities/DTO/ServiceCatalog/ServiceResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ecommerce.Entities.DTO.ServiceCatalog
{
    public class ServiceResponse
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public string ImagesUrl { get; set; }
        public string WebsiteUrl { get; set; }
        public string CategoryName { get; set; }
        public string ProviderId { get; set; }
        public string Status { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}
